### PR TITLE
fix typo error

### PR DESCRIPTION
**Pull request by hougangliu**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/katib/280)

### CODE REVIEW

File: examples/MinikubeDemo.md [hunk 1]
```diff
@@ -144,7 +144,7 @@ Let's use [this](./workerConfigMap.yaml) template.
 ```
 kubectl apply -f workerConfigMap.yaml
 ```
-This template will share among blow three demos(Grid, Hyperband, and GPU).
+This template will share among below three demos(Grid, Hyperband, and GPU).
```
Comment (richardsliu) [inline examples/MinikubeDemo.md]: "This template will be shared by the three demos below (Grid, Hyperband, and GPU).

Comment (hougangliu) [reply examples/MinikubeDemo.md]: thanks!
Updated

**Pull request merged**